templates/FFV_form_factors.cpp.in

```diff
@@ -501,16 +501,19 @@ double Kallenlambda(double m1, double m2, double m3)
 
 double Kallenlambdasqrt(double m1, double m2, double m3)
 {
-    return std::sqrt(Power2(m1) + Power2(m2) + Power2(m3) - 2.*m1*m2 - 2.*m2*m3 - 2.*m3*m1);
+    return std::sqrt(Kallenlambda(m1,m2,m3));
 }
 
 double DiscontinuityB(double m12, double m2, double m3)
 {
-    return Kallenlambdasqrt(m12,Power2(m2),Power2(m3)) 
-	    * std::log((Power2(m2)+Power2(m3)-m12+Kallenlambdasqrt(m12,Power2(m2),Power2(m3)))/(2.*m2*m3)) / m12;
+    double m12 = m12;
```

Comment:
Sorry, but these three variable definitions are redundant. Please remove them. :)

Reply:
Thanks for spotting this, thought I had corrected that.  Reverted now.